docs/source/installation-guide.rst

```diff
@@ -104,16 +104,15 @@ QUICK CPU versions can be configured and built as follows:
    * GNU compiler (default): ``-DCOMPILER=GNU``
    * Intel compiler: ``-DCOMPILER=INTEL``
    * Clang compiler: ``-DCOMPILER=CLANG``
+   Note that requesting the Clang compiler requires the GNU Fortran compiler (`gfortran`) to be installed. C/C++ code will be compiled by Clang, 
+   while Fortran code will be compiled by gfortran.
 
-3. Build and install::
+2. Build and install::
 
-	make -j$(sysctl -n hw.ncpu)  # on macOS
-	# or
-	make -j$(nproc)              # on Linux
+	make -j
```

Comment:
You could add a note that the flag `-j` enables parallel builds that use all available CPU cores for compilation.